code/game/objects/structures/flora.dm

```diff
@@ -314,10 +315,117 @@
 	..()
 	unhide_user(user)
 
+/obj/item/kirbyplants/plant1
+	icon_state = "plant-1"
+
+/obj/item/kirbyplants/plant2
+	icon_state = "plant-2"
+
+/obj/item/kirbyplants/plant3
+	icon_state = "plant-3"
+
+/obj/item/kirbyplants/plant4
+	icon_state = "plant-4"
+
+/obj/item/kirbyplants/plant5
+	icon_state = "plant-5"
+
+/obj/item/kirbyplants/plant6
+	icon_state = "plant-6"
+
+/obj/item/kirbyplants/plant7
+	icon_state = "plant-7"
+
+/obj/item/kirbyplants/plant8
+	icon_state = "plant-8"
+
+/obj/item/kirbyplants/plant9
+	icon_state = "plant-9"
+
+/obj/item/kirbyplants/plant10
+	icon_state = "plant-10"
+
+/obj/item/kirbyplants/plant11
+	icon_state = "plant-11"
+
+/obj/item/kirbyplants/plant12
+	icon_state = "plant-12"
+
+/obj/item/kirbyplants/plant13
+	icon_state = "plant-13"
+
+/obj/item/kirbyplants/plant14
+	icon_state = "plant-14"
+
+/obj/item/kirbyplants/plant15
+	icon_state = "plant-15"
+
+/obj/item/kirbyplants/plant16
+	icon_state = "plant-16"
+
+/obj/item/kirbyplants/plant17
+	icon_state = "plant-17"
+
+/obj/item/kirbyplants/plant18
+	icon_state = "plant-18"
+
+/obj/item/kirbyplants/plant19
+	icon_state = "plant-19"
+
+/obj/item/kirbyplants/plant20
+	icon_state = "plant-20"
+
+/obj/item/kirbyplants/plant21
+	icon_state = "plant-21"
+
+/obj/item/kirbyplants/plant22
+	icon_state = "plant-22"
+
+/obj/item/kirbyplants/plant23
+	icon_state = "plant-23"
+
+/obj/item/kirbyplants/plant24
+	icon_state = "plant-24"
+
+/obj/item/kirbyplants/plant25
+	icon_state = "plant-25"
+
+/obj/item/kirbyplants/plant26
+	icon_state = "plant-26"
+
+/obj/item/kirbyplants/plant27
+	icon_state = "plant-27"
+
+/obj/item/kirbyplants/plant28
+	icon_state = "plant-28"
+
+/obj/item/kirbyplants/plant29
+	icon_state = "plant-29"
+
+/obj/item/kirbyplants/plant30
+	icon_state = "plant-30"
+
+/obj/item/kirbyplants/plant31
+	icon_state = "plant-31"
+
+/obj/item/kirbyplants/plant32
+	icon_state = "plant-32"
+
+/obj/item/kirbyplants/plant33
+	icon_state = "plant-33"
+
+/obj/item/kirbyplants/plant34
+	icon_state = "plant-34"
+
+/obj/item/kirbyplants/applebush
+	icon_state = "applebush"
+
 /obj/item/kirbyplants/dead
+	icon_state = "plant-dead"
```

Comment:
why diverge from the numbered scheme earlier? Is there something particularly different about this kirbyplants that set them apart from the rest?

Reply:
The apple bush was never as random as a dead plant